pkg/stanza/fileconsumer/file_windows.go

```diff
@@ -17,5 +17,6 @@ func (m *Manager) preConsume(_ context.Context, _ []*reader.Reader) {
 // On windows, we close files immediately after reading because they cannot be moved while open.
 func (m *Manager) postConsume(readers []*reader.Reader) {
 	m.previousPollFiles = readers
+	m.openFiles.Reset(m.ActiveFiles()...)
```

Comment:
A couple things don't seem right about this.

We've removed the parameter for non-windows but left it here.

Active files should not move directly to open files, but to previous poll files. I'm not sure we should be changing open files at all here.